pkg/member/member_control.go

```diff
@@ -258,11 +258,17 @@ func (m *memberControl) doPromoteMember(ctx context.Context, member *etcdserverp
 		m.logger.Info("Member promoted ", member.Name, " : ", member.ID)
 		return nil
 	} else if errors.Is(err, rpctypes.Error(rpctypes.ErrGRPCMemberNotLearner)) { //Member is not a learner
-		if member.PeerURLs[0] == "http://localhost:2380" { //[]string{"http://localhost:2380"}[0] {
+		if member.PeerURLs[0] == "http://localhost:2380" {
```

Comment:
The name of the interface should be `MemberControl` and not `ControlMember`. The latter is a verb and is more apt for a function name and not for an interface. Also the struct is named as `memberControl`

Reply:
Updated the interface name to `Control` after an ad-hoc consultation